src/main/java/sirius/pasta/tagliatelle/Tagliatelle.java

```diff
@@ -124,15 +125,29 @@
     }
 
     private List<TemplateExtension> loadExtensions(String target) {
-        return Sirius.getClasspath()
-                     .find(Pattern.compile("(default/|customizations/[^/]+/)?extensions/"
-                                           + Pattern.quote(target)
-                                           + "/.*.pasta"))
-                     .map(m -> m.group(0))
-                     .map(this::resolveToTemplateExtension)
-                     .filter(Objects::nonNull)
-                     .sorted()
-                     .collect(Collectors.toList());
+        List<TemplateExtension> extensionsForTarget = Sirius.getClasspath()
+                                                            .find(Pattern.compile("(customizations/[^/]+/)?extensions/"
+                                                                                  + Pattern.quote(target)
+                                                                                  + "/.*.pasta"))
+                                                            .map(m -> m.group(0))
+                                                            .map(this::resolveToTemplateExtension)
+                                                            .filter(Objects::nonNull)
+                                                            .toList();
+
+        // Also check for extensions provided as default - which are only loaded, if no overwrites (same path without
+        // /default/ in front) exists...
+        return Streams.concat(extensionsForTarget.stream(),
+                              Sirius.getClasspath()
+                                    .find(Pattern.compile("default/extensions/" + Pattern.quote(target) + "/.*.pasta"))
```

Comment:
s.a.

Reply:
this isn't constant as "target" is passed in - however the result of the method call is cached...